src/docs/get-started/install/_ios-setup.md

```diff
@@ -113,11 +113,16 @@ You'll also need to set up physical device deployment in Xcode.
         ios/Runner.xcworkspace` in a terminal window from your Flutter project
         directory.
```

Comment:
1. Select the device you intend to deploy to in the device drop-down menu next to the run button.